docs/DeveloperGuide.md

```diff
@@ -591,42 +633,322 @@ testers are expected to do more *exploratory* testing.
+      Then, enter `find n/e`, and enter `view 1`.
+
+   2. **Test Case**: `edit 1 n/EditTest4`
+      **Expected Outcome**: On the right side panel, the profile is updated such that the name is now "EditTest4". Similarly, the name should be the same on the left panel.
+
+   3. **Test Case**: `edit 1 n/Void`
+      **Expected Outcome**: On the left side panel, the edited employee disappears. The right side panel either shows the next employee (if there is any), or shows the home panel.
+
+### Deleting an employee
+
+1. Deleting an employee in general:
+
+   1. **Prerequisites**: Ensure that there is at least 1 employee in the database.
+
+   2. **Test Case**: `delete 1`<br>
+      **Expected Outcome**: First employee is deleted from the list. Details of the deleted employee shown in the status message.
+      Right panel now shows the next employee (if there is any), or shows the home panel.
+
+   3. **Test Case**: `delete 0`<br>
+      **Expected Outcome**: No changes occur. An error message appears, saying that the format of the command entered is incorrect.
+
+   4. Other incorrect delete commands to try: `delete`, `delete x` (where x is larger than the list size), `delete abcdef` <br>
+      **Expected Outcome**: Similar to previous.
+
+### Batch-adding employees
+
+This feature requires the use of csv files. To follow the subsequent tests, create and save two csv files with the contents as given below.
+
+**1st csv file**: save as `data/success.csv`.
+
+```csv
+Name,Phone,Email,Position,Department,Address,Leave,Tags,
+Kim Meier,84824249,kimmeier@example.com ,Frontend Engineer,Information Technology,Little India,20,Promotion coming,
+Petris Mueller,96722343,,Marketing Intern,Marketing,,13,,
+Paul Morty,,paul@example.com,UI/UX Engineer,Sales,,,Innovation Lead,
+```
+
+**2nd csv file**: save as `data/duplicatePerson.csv`.
+
+```csv
+Name,Phone,Email,Position,Department,Address,Leaves,Tags
+Prittam Ravi,87438807,prittam@example.com,Chief Technology Officer,General Management,Blk 30 Lorong 3 Serangoon Gardens #07-18,14,collegaues/friends
+Peter Lim,99272758,peter@example.com,Accountant,Finance,Blk 30 Lorong 3 Serangoon Gardens #07-18,14,collegaues/friends
+Shawn Kok,99272758,shawn@example.com,Recruiter,Human Resources,Blk 30 Lorong 3 Serangoon Gardens #07-18,14,collegaues/friends
+Kevin Chang,99272758,kevin@example.com,Factory Worker,Production,Blk 30 Lorong 3 Serangoon Gardens #07-18,14,collegaues/friends
+Ng Shi Jun,99272758,shijun@example.com,Frontend Developer,Information Technology,Blk 30 Lorong 3 Serangoon Gardens #07-18,14,collegaues/friends
+Prittam Ravi,87438807,prittam@example.com,Chief Technology Officer,General Management,Blk 30 Lorong 3 Serangoon Gardens #07-18,14,collegaues/friends
+```
+
+1. Batch-adding multiple employees:
+
+   1. Prerequisites: Must not have used any of the names provided in `data/success.csv`.
```

Comment:
can add no commas in the fields within the CSV file

Reply:
Ordering of the columns need to follow the one provided

Reply:
Yep will add.